genesis/cmd/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/deployer/pipeline"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/deployer/state"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/foundry"
+	"github.com/ethereum-optimism/optimism/op-service/ioutil"
+	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum-optimism/supersim/genesis/worldgen"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/joho/godotenv"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	// Load the .env file
+	err := godotenv.Load()
```

Comment:
I think this will make working with docker or deployed backend environments harder. On a code level it might make sense for it to expect the env vars to be set already, then for local develop we can use dotenv as part of the builds steps.

Reply:
oh wait that's copy pasta it will be removed I don't use envvars

Reply:
great catch